pkg/controller/account/account_controller.go

```diff
@@ -643,7 +655,13 @@ func (r *ReconcileAccount) assumeRole(
 
 	// The role ARN made up of the account number and the role which is the default role name
 	// created in child accounts
-	var roleArn = fmt.Sprintf("arn:aws:iam::%s:role/%s", currentAcctInstance.Spec.AwsAccountID, roleToAssume)
+	var roleArn string
+	roleArn = fmt.Sprintf("arn:aws:iam::%s:role/%s", currentAcctInstance.Spec.AwsAccountID, roleToAssume)
+
+	// if Fedramp change the role ARN
+	if config.IsFedramp() {
+		roleArn = fmt.Sprintf("arn:aws-us-gov:iam::%s:role/%s", currentAcctInstance.Spec.AwsAccountID, roleToAssume)
+	}
```

Comment:
would it be possible to encapsulate the roleArn(s) in a similar way as the default regions?